src/compiler/utilities.ts

```diff
@@ -1281,8 +1282,8 @@ export function getInternalEmitFlags(node: Node): InternalEmitFlags {
 export type ScriptTargetFeatures = ReadonlyMap<string, ReadonlyMap<string, string[]>>;
 
 /** @internal */
-export function getScriptTargetFeatures(): ScriptTargetFeatures {
-    return new Map(Object.entries({
+export const getScriptTargetFeatures: () => ScriptTargetFeatures = /* @__PURE__ */ memoize(() =>

```

Comment:
I don't think we use this annotation anywhere else - any reason you felt we needed it?

Reply:
I had suggested sticking it on the closure inside of the memoize call but they put it here instead (which is roughly equivalent, I guess). Mainly it's that this type is an aliased ReadonlyMap. Not sure why it's named, per se

Reply:
I just tacked a small change onto the PR to move the annotation to the other place; wanting to merge this!